jetty-util/src/test/java/org/eclipse/jetty/util/security/PasswordTest.java

```diff
@@ -44,4 +56,35 @@ public void testObfuscateUnicode()
         String obfuscate = Password.obfuscate(password);
         assertEquals(password, Password.deobfuscate(obfuscate));
     }
+
+    @Test
+    public void testCommandLineUsage() throws IOException, InterruptedException
+    {
+        ProcessBuilder passwordBuilder = new ProcessBuilder()
+            .directory(MavenTestingUtils.getTargetDir())
+            .command("java",
+                "-cp", MavenTestingUtils.getTargetPath("classes").toString(),
+                Password.class.getName(),
+                "user", "password")
+            .redirectErrorStream(true);
+
+        Process passwordProcess = passwordBuilder.start();
+        try (InputStreamReader inputStreamReader = new InputStreamReader(passwordProcess.getInputStream());
+             BufferedReader reader = new BufferedReader(inputStreamReader))
+        {
+            String output = reader.lines().collect(Collectors.joining(System.lineSeparator()));
+            if (passwordProcess.waitFor(5, TimeUnit.SECONDS))
+            {
+                int exitCode = passwordProcess.exitValue();
+                assertThat("Non-error exit code: " + output, exitCode, is(0));
+                assertThat("Output", output, not(containsString("Exception")));
```

Comment:
I would add an assert for minimal expected output (e.g. the original password is printed back), something like `assertThat(output, containsString("password"));`.

Reply:
will do

Reply:
Done.